core/src/main/java/com/scalar/db/service/TransactionService.java

```diff
@@ -57,34 +54,40 @@ public DistributedTransaction start(String txId) throws TransactionException {
     return manager.start(txId);
   }
 
+  @Deprecated
```

Comment:
If I'm not wrong, this is a public interface so adding a comment would be very helpful to the user to know what he should be using instead.
https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html
For example : 
```
/**
 * @deprecated  As of release 1.3, replaced by {@link #getPreferredSize()}
 */
 ```